cmake-modules/add_nlohmann_json.cmake

```
@@ -8,15 +8,15 @@
 
 # Storage requires 3.6.0 version for using `contains` feature
 # Using QUIET to avoid warning message if lib is not found
-find_package(nlohmann_json 3.6.0 EXACT QUIET)
+find_package(nlohmann_json 3.6.0 QUIET)
```

We should use the newest available to get the bug fixes, such as:
https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent/releases/tag/v3.6.1

```suggestion
find_package(nlohmann_json 3.9.1 QUIET)
```